module-8/M8/server/config/db.js

```diff
@@ -0,0 +1,7 @@
+import mongoose from 'mongoose';
+import { Sequelize } from 'sequelize';
+
+const sequelize = new Sequelize('postgres://db_user:135256@localhost:5432/portfolio' , {dialect: 'postgres', logging: false});
```

Comment:
Database usernames/passwords should not be hardcoded.
We want to get database values from environment variables.